tests/DCPS/SkipSerialize/DataReaderListener.cpp

```diff
@@ -68,29 +68,31 @@ void DataReaderListenerImpl::on_data_available(DDS::DataReader_ptr reader)
           valid_ = false;
         }
 
-        std::cout << "Message: subject    = " << message.subject.in() << std::endl
-                  << "         subject_id = " << message.subject_id   << std::endl
-                  << "         from       = " << message.from.in()    << std::endl
-                  << "         count      = " << message.count        << std::endl
-                  << "         text       = " << message.text.in()    << std::endl;
-
-        if (std::string("Comic Book Guy") != message.from.in() &&
-            std::string("OpenDDS-Java") != message.from.in()) {
-          std::cout << "ERROR: Invalid message.from" << std::endl;
-          valid_ = false;
-        }
-        if (std::string("Review") != message.subject.in()) {
-          std::cout << "ERROR: Invalid message.subject" << std::endl;
-          valid_ = false;
-        }
-        if (std::string("Worst. Movie. Ever.") != message.text.in()) {
-          std::cout << "ERROR: Invalid message.text" << std::endl;
-          valid_ = false;
-        }
-        if (message.subject_id != 99) {
-          std::cout << "ERROR: Invalid message.subject_id" << std::endl;
-          valid_ = false;
+        std::cout << "Message:" << std::endl
+                  << "         count      = " << message.count        << std::endl;
+        for (CORBA::ULong i = 0; i < message.serialized_data.length(); i++)
+        {
+          std::cout << "         data[" << i << "] = " << message.serialized_data[i] << std::endl;
+
         }
+        //TODO create check for new message similar to commented below
```

Comment:
[The LargeSample test does something like that for an OctetSeq](https://github.com/objectcomputing/OpenDDS/blob/c9b8469ee8022ca581573f863fc12f8e142ca4b5/tests/DCPS/LargeSample/DataReaderListener.cpp#L101-L116).

Reply:
> [The LargeSample test does something like that for an OctetSeq](https://github.com/objectcomputing/OpenDDS/blob/c9b8469ee8022ca581573f863fc12f8e142ca4b5/tests/DCPS/LargeSample/DataReaderListener.cpp#L101-L116).

I think my update captures the essence of what you were showing.